docs/specification.md

```diff
@@ -166,12 +185,14 @@ Here's an example of a sequence collection organized into the canonical seqcol o
 This object would validate against the JSON Schema above.
 The object is a series of arrays with matching length (`3`), with the corresponding entries collated such that the first element of each array corresponds to the first element of each other array.
 For the rationale why this structure was chosen instead of an array of annotated sequences, see [*Footnote F1*](#f1-why-use-an-array-oriented-structure-instead-of-a-sequence-oriented-structure).
-Implementations `MUST` provide at least the structure specified in this schema.
+The implementation `MUST` define its structure in a JSON Schema, such as the example schema defined in step 1.
 Implementations `MAY` choose to extend this schema by adding additional attributes.
+Implmentations `MAY` also use a schema, but we `RECOMMEND` the schema extend the base schema defined above.
 This schema extends vanilla JSON Schema in two ways; first, it provides the `collated` qualifier.
```

Comment:
Lines 191-195 relates to the schema (but not, I believe, duplicated content) and should be moved/merged with section 1